lib/run.js

```diff
@@ -13,6 +13,10 @@ const reInterval = require('reinterval')
 const histAsObj = histUtil.histAsObj
 const addPercentiles = histUtil.addPercentiles
 
+// TODO upstream to hdr-histogram-percentiles-obj?
+histUtil.percentiles.push(2.5, 97.5)
+histUtil.percentiles.sort((a, b) => a - b)
```

Comment:
I think those needs to be upstreamed.

Reply:
Aye: https://github.com/thekemkid/hdr-histogram-percentiles-obj/pull/2 May have to change the 2.5% name (as mentioned in the PR)